pkg/trace/agent/agent.go

```diff
@@ -246,34 +277,11 @@ func (a *Agent) Process(p *api.Payload) {
 
 		// Extra sanitization steps of the trace.
 		for _, span := range chunk.Spans {
-			for k, v := range a.conf.GlobalTags {
```

Comment:
I'm fine with the other helper function being pulled out, since it was previously it's own `{ }` block, so a different function is reasonable.
But this one seems okay to just leave as-is for now. It doesn't affect this change anyway, so let's leave changes that aren't required until later.

Reply:
This function is also in its own `{ }` block.